src/components/Projects/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import maskGroup from "../images/mask_group.svg";
+import projects from "../../Data";
+import "./projects.css";
+
+function Projects() {
+  return (
+    <div className="projects">
+      <h1>Projetos</h1>
+      {projects.map((project, index) => (
+        <div className={`project-${index} project`} key={project.title}>
```

Comment:
You should not add the index to your css class. You can handle the project direction determining if its index is even or odd.